src/status_im2/subs/onboarding.cljs

```diff
@@ -48,6 +50,22 @@
  (fn [[intro-wizard multiaccounts]]
    (recover/existing-account? (:root-key intro-wizard) multiaccounts)))
 
+(re-frame/reg-sub
+ :multiaccounts/login-profiles-picture
+ :<- [:multiaccounts/multiaccounts]
+ :<- [:mediaserver/port]
+ (fn [[multiaccounts port] [_ target-key-uid]]
+   (let [image-name (-> multiaccounts
+                        (get-in [target-key-uid :images])
+                        first
+                        :type)]
```

Comment:
image name stored in status-go is the `:type` of images, `"large"`, `"thumbnail"`